tenable_io/helpers/workbench.py

```diff
@@ -0,0 +1,417 @@
+        file_id = self._client.workbenches_api.export_request(
+            WorkbenchesApi.FORMAT_NESSUS,
+            WorkbenchesApi.REPORT_VULNERABILITIES,
+            WorkbenchesApi.CHAPTER_VULN_BY_ASSET,
+            date_range=date_range,
+            plugin_id=plugin_id,
+            filters=[{
+                'quality': 'qt',
+                'filter': 'severity',
+                'value': 'All',
+            }],
+        )
+
+        wait_until(lambda: self._client.workbenches_api.export_status(file_id) == WorkbenchesApi.STATUS_EXPORT_READY)
+
+        iter_content = self._client.workbenches_api.export_download(file_id)
+
+        with tempfile.NamedTemporaryFile() as temp:
+
+            for chunk in iter_content:
+                temp.write(chunk)
+                temp.flush()
+
+            assets = []
+            try:
+                gen = WorkbenchParser.parse(temp.name)
+                while True:
+                    report = next(gen)
+                    assets.append(AssetVulnerabilities().from_report(report))
+                    if page_size and len(assets) >= page_size:
+                        yield assets
+                        assets = []
+                pass
+            except StopIteration:
+                pass
+
+        if len(assets) > 0:
+            yield assets
+
+    def vulnerabilities(self, date_range=1, asset_id=None, page_size=DEFAULT_PAGE_SIZE):
+        """Retrieve recorded vulnerabilities from the workbench nessus report.
+
+        :param date_range: The number of days of data prior to today to return, default to 1.
+        :param asset_id: If specified, returns only vulnerabilities for the asset identified by asset_id, default to
+            None.
+        :param page_size: The page size of the pages returns by the iterator, default to DEFAULT_PAGE_SIZE.
+        :raise TenableIOApiException:  When API error is encountered.
+        :return: Iterator that yields pages of :class:`Vulnerability`.
+        """
+        return self.vulnerabilities_parse(date_range, asset_id, page_size)
+
+    def vulnerabilities_api(self, date_range=1, asset_id=None):
+        """Retrieve recorded vulnerabilities.
+
+        :param date_range: The number of days of data prior to today to return, default to 1.
+        :param asset_id: If specified, returns only vulnerabilities for the asset identified by asset_id, default to
+            None.
+        :raise TenableIOApiException:  When API error is encountered.
+        :return: A list of :class:`Vulnerability`.
+        """
+        if asset_id is not None:
+            vulnerabilities = self._client.workbenches_api.asset_vulnerabilities(
+                asset_id, date_range=date_range).vulnerabilities
+        else:
+            vulnerabilities = self._client.workbenches_api.vulnerabilities(date_range=date_range).vulnerabilities
+        return vulnerabilities
+
+    def vulnerabilities_parse(self, date_range=1, asset_id=None, page_size=DEFAULT_PAGE_SIZE):
```

Comment:
I'm seeing some parsing errors throw here for one of my accounts. 

`xml.etree.ElementTree.ParseError: not well-formed (invalid token): line 87553, column 108`